exonum/src/api/backends/actix.rs

```diff
@@ -34,15 +35,14 @@ use std::{
     net::SocketAddr,
     result,
     str::FromStr,
-    sync::{mpsc, Arc},
+    sync::Arc,
     thread::{self, JoinHandle},
 };
 
+use crate::api::manager::UpdateEndpoints;
 use crate::api::{
-    error::Error as ApiError,
-    manager::{ApiManager, RestartServer},
-    ApiAccess, ApiAggregator, ApiBackend, ApiScope, ExtendApiBackend, FutureResult, Immutable,
-    Mutable, NamedWith, Result,
+    self, manager::ApiManager, ApiAccess, ApiAggregator, ApiBackend, ApiScope, ExtendApiBackend,
```

Comment:
```suggestion
    self, manager::{ApiManager, UpdateEndpoints}, ApiAccess, ApiAggregator, ApiBackend, ApiScope, ExtendApiBackend,
```
This change allows to get rid of 42 line.